domains/tio.json

```diff
@@ -0,0 +1,17 @@
+{
+    "repo": "https://github.com/BOTCAHX/myporto",
+
+    "owner": {
+        "username": "hostinger-bot",
+        "email": "hostingerbalance@gmail.com"
+    },
+
+    "target": {
+        "CNAME": {
+            "name": "tio",
+            "value": "myporto15.netlify.app."
```

Comment:
- We do not support netlify
- There is a period at the end of your CNAME value which is invalid.